### PR TITLE
SOC-664 Provide a text version of the email.

### PR DESCRIPTION
**Pull request by garthwebb**
Strip the tags and eliminate extra whitespace

https://wikia-inc.atlassian.net/browse/SOC-664

### CODE REVIEW

File: extensions/wikia/Email/EmailController.class.php [hunk 1]
```diff
@@ -232,6 +239,24 @@ protected function getBody() {
...
+	protected function bodyHtmlToText( $html ) {
+		$bodyText = strip_tags( $html );
+
+		// Get rid of multiple blank white lines
+		$bodyText = preg_replace( '/^\h*\v+/m', '', $bodyText );
+
+		// Get rid of leading spacing/indenting
+		$bodyText = preg_replace( '/^[\t ]+/m', '', $bodyText );
+		return $bodyText;
+	}
```
Comment (armonr) [inline extensions/wikia/Email/EmailController.class.php]: This method can use a test :)

**Comment by armonr** [comment]
One comment, otherwise, :ship: it

**Comment by jsutterfield** [comment]
LGTM

File: extensions/wikia/Email/EmailController.class.php (hunk 1 above)
Comment (garthwebb) [inline extensions/wikia/Email/EmailController.class.php]: Eh, I'm not convinced.  There really isn't anything in the acceptance criteria about how this happens or looks except 'no HTML.'  I'm not a big fan of tests where a failing test gets fixed simply by copying and pasting the new output to the accepted results of the test.

**Pull request merged**